### PR TITLE
feat: Integrate SVO plugin to VEP - add PC field

### PR DESCRIPTION
**Pull request by diegomscoelho**
Jira: [Card](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-5028)

## Description

[StructuralVariationOverlap](https://github.com/Ensembl/VEP_plugins/blob/release/107/StructuralVariantOverlap.pm) plugin retrieves information from overlapping structural variants. Output should have percentage (PC) + allele frequency (AF) if exists. Using vep `--custom`, we can already retrieve AF information easily, but percentage will not be calculated. This PR is meant to add PC information by default into SV custom overlap usage.

## Test

1) Use `--custom` with and without `PC` as a field. For example:
```sh
vep ... --custom gnomad_chr22.vcf.gz,prefix,vcf,overlap,0,PC,info_columns
vep ... --custom gnomad_chr22.vcf.gz,prefix,vcf,overlap,0,info_columns
```

### CODE REVIEW

File: INSTALL.pl [hunk 1]
```diff
@@ -260,9 +260,9 @@ sub read_config_from_environment {
 ## Set the indexed cache url if it's been overwritten by the user
 $CACHE_URL_INDEXED = $CACHE_URL;
 
-$CACHE_URL  ||= "ftp://ftp.ensembl.org/pub/release-$DATA_VERSION/variation/vep";
-$CACHE_URL_INDEXED  ||= "ftp://ftp.ensembl.org/pub/release-$DATA_VERSION/variation/indexed_vep_cache";
-$FASTA_URL  ||= "ftp://ftp.ensembl.org/pub/release-$DATA_VERSION/fasta/";
+$CACHE_URL  ||= "https://ftp.ensembl.org/pub/release-$DATA_VERSION/variation/vep";
```
Comment (dglemos) [inline INSTALL.pl:263]: The update from ftp to https is already merged into release/109: https://github.com/Ensembl/ensembl-vep/blob/release/109/INSTALL.pl#L263

Comment (diegomscoelho) [reply INSTALL.pl:263]: That's true. I still don't know why github it's not identifying those alterations on `postreleasefix/109` and marking as an alteration to code.

Comment (diegomscoelho) [reply INSTALL.pl:263]: I think I got what I did. I recommitted last commit from postrelasefix/109 (https://github.com/Ensembl/ensembl-vep/pull/1279/commits/a99f64f1202f1ad67d07a939809bb4cb87426823) and it's treating as an alteration

**Review by nuno-agostinho — APPROVED** [review]
Hey @diegomscoelho, everything seems fine, thanks!

We should update our documentation to mention the use of PC in `--custom` and deprecate the SVO plugin for 109.

**Comment by nuno-agostinho** [comment]
Merged to `release/109` and `main`.

**Pull request merged**